src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -679,6 +686,9 @@ powerflex_storage_pool=pool1
 # 5 nodes model is not supported yet.
 powerflex_cluster_config=3_node
 
+# Interface used for configuration by Ansible. Added for RHOSP 16 as the ctlplane ip is not blocked.
+powerflex_config_interface=eno3
```

Comment:
What node is this referring to ? Should probably not be a setting regardless

Reply:
This is the interface that Ansible will use to connect to each of the nodes when deploying PowerFlex. This interface must be accessible from the howt where the ansible playbook are executed, aka the Director. 
When setting up the facts for each node, Ansible will fetch the IP bound to this interface and will use it from there to execute commands.

Reply:
So this is the provisioning interface on each of the nodes ? This is unlikely to work , since we can have different nics associated with different nodes.

Reply:
In that case, that would be an issue. We'll need to get the IP based upon something not static then.

Reply:
Fixed. Ansible get the IP from first host included in the mdms group in the inventory file. Will remove this reference